lhotse/cut.py

```diff
@@ -4067,33 +4067,53 @@ def extend_by(
             for cut in self
         )
 
+    @staticmethod
+    def num_windows(sig_len: Seconds, win_len: Seconds, hop: Seconds) -> int:
```

Comment:
Can we rename it to `compute_num_windows` and move it to `lhotse.utils` as a standalone function? It looks like it could come useful in other contexts too.